packages/db/src/translate/reconciliation_reports.ts

```diff
@@ -0,0 +1,23 @@
+import { PostgresReconciliationReportRecord } from '../types/reconciliation_report';
+
+/**
+ * Generate an API Reconciliation Report record from a PostgreSQL record.
+ *
+ * @param {Object} pgReconciliationReport - a PostgreSQL reconciliation report record
+ * @returns {Object} an API reconciliation report record
+ */
+export const translatePostgresReconReportToApiReconReport = (
+  pgReconciliationReport: PostgresReconciliationReportRecord
+) => {
+  const apiReconciliationReport = {
+    // id or cumulus_id?
+    name: pgReconciliationReport.name,
+    type: pgReconciliationReport.type,
+    status: pgReconciliationReport.status,
+    location: pgReconciliationReport.location,
+    error: pgReconciliationReport.error,
```

Comment:
We can simplify the statements with:
...pick(pgReconciliationReport, ['name', 'type', 'status', 'location', 'error']),

Reply:
Ok, that is helpful, but makes the file pretty inconsistent with all the other translate files (none of the ones I looked at used pick)